android/app-newm/src/main/java/io/newm/screens/library/NFTLibraryPresenter.kt

```diff
@@ -58,22 +63,22 @@ class NFTLibraryPresenter(
             } else {
                 flowOf()
             }
-        }.collectAsState(initial = emptyList())
+        }.collectAsRetainedState(initial = emptyList())
```

Comment:
Interesting, so it was not working as expected because we were using `collectAsState` as opposed to `collectAsRetainedState`?